src/Microsoft.DotNet.Interactive.Documents/InteractiveDocument.cs

```diff
@@ -65,7 +65,7 @@ public async IAsyncEnumerable<InteractiveDocument> GetImportsAsync(bool recursiv
     {
         EnsureImportFieldParserIsInitialized();
 
-        if (!TryGetKernelInfoFromMetadata(Metadata, out var kernelInfos))
+        if (!TryGetKernelInfosFromMetadata(out var kernelInfos))
```

Comment:
minor nit: This call uses the public instance overload while others below use the static overload. Could we standardize to the same call?